### PR TITLE
feat: refactor encrypt and decrypt + CLI examples

### PR DESCRIPTION
**Pull request by jrschumacher**
- Refactor encrypt and decrypt
- Docs
  - Encrypt, Decrypt
  - Auth

### CODE REVIEW

File: docs/man/auth/print-access-token.md [hunk 1]
```diff
@@ -9,4 +9,13 @@ command:
       default: false
 ---
 
-Retrieves a new OIDC Access Token using the client credentials from the OS-specific keychain and prints to stdout if found.
+> [!NOTE]
+> Requires experimental profiles feature.
+>
+> | OS | Keychain | State |
+> | --- | --- | --- |
+> | macOS | Keychain | Stable |
```
Comment (jrschumacher) [inline docs/man/auth/print-access-token.md:17]:
```suggestion
> | MacOS | Keychain | Stable |
```

File: docs/man/encrypt/_index.md [hunk 2]
```diff
@@ -14,34 +14,84 @@ command:
...
+restrict access to the data based on entity entitlements.
+
+```shell
+# output to hello.txt.tdf with attribute
+otdfctl encrypt hello.txt --out hello.txt.tdf --attr https://example.com/attr/attr1/value/value1
+```
+
+## NanoTDF
+
+NanoTDF is a lightweight TDF format that is more performant and has a smaller footprint than TDF3. NanoTDF does not
```
Comment (jakedoublev) [inline docs/man/encrypt/_index.md:79]:
```suggestion
NanoTDF is a lightweight TDF format that is more performant and has a smaller footprint than ZTDF. NanoTDF does not
```

File: pkg/handlers/tdf.go [hunk 3]
```diff
@@ -17,115 +17,154 @@ var (
...
-	}
-	switch {
-	case bytes.Equal([]byte("PK"), toInspect[0:2]):
-		return h.InspectZTDF(toInspect)
-	case bytes.Equal([]byte("L1L"), toInspect[0:3]):
-		return h.InspectNanoTDF(toInspect)
+func (h Handler) DecryptBytes(toDecrypt []byte, disableAssertionCheck bool) (*bytes.Buffer, error) {
+	out := &bytes.Buffer{}
+	pt := io.Writer(out)
+	ec := bytes.NewReader(toDecrypt)
```
Comment (jakedoublev) [inline pkg/handlers/tdf.go:97]: what are `pt` and `ec` short for here?

**Review by jakedoublev — APPROVED** [review]

File: pkg/handlers/tdf.go (hunk 3 above)
Comment (jrschumacher) [reply pkg/handlers/tdf.go:97]: plaintext and encrypted content

**Review by jakedoublev — APPROVED** [review]

File: pkg/handlers/tdf.go (hunk 3 above)
Comment (jrschumacher) [reply pkg/handlers/tdf.go:97]: In the future I might just do `r` and `w` so its less specific to the op and more to the IO

**Pull request merged**